src/extensions/common/NiceTable.tsx

```diff
@@ -3,9 +3,10 @@ import { Delete, Edit } from "@material-ui/icons";
 import React, { FunctionComponent } from 'react';
 import './NiceTable.css';
 
-type Row = {
+interface Row {
```

Comment:
There were two versions of NiceTable (in two separate places in the source tree), and so we just combined them in the extensions/common directory